lib/pages/settings_emotes/settings_emotes.dart

```diff
@@ -343,32 +340,10 @@ class EmotesSettingsController extends State<EmotesSettings> {
 
         if (output == null) return;
 
-        if (kIsWeb || PlatformInfos.isMobile) {
-          await Share.shareXFiles(
-            [XFile(fileName, bytes: Uint8List.fromList(output))],
-          );
-        } else {
-          String? savePath = await FilePicker.platform
-              .saveFile(fileName: fileName, allowedExtensions: ['zip']);
-
-          if (savePath == null) {
-            // workaround for broken `xdg-desktop-portal-termfilechooser`
```

Comment:
Can you please include this workaround then 🤔 ? It's otherwise completely broken for my use case 😂 ... Yes, I know, workarounding other packages bugs is always annoying but it's super useful to me.

Reply:
If this is broken, then also save all files is broken. So why fixing it at one place but not the other? Btw I could not reproduce it and I'm still able to save files under Linux.